src/kex_lwe_frodo/kex_lwe_frodo_macrify.c

```diff
@@ -13,7 +13,7 @@ OQS_KEX *MACRIFY(OQS_KEX_lwe_frodo_new)(OQS_RAND *rand, const uint8_t *seed, con
 
 	k = malloc(sizeof(OQS_KEX));
 	if (k == NULL) {
-		goto err;
+		return NULL;
```

Comment:
This change isn't essential. Both goto/return would work either well.

Reply:
I'd rather keep this as goto err.  While it's currently equivalent and no cleanup is required, if some later edit adds code above this line, they may no longer be equivalent, and changing it back to goto err might get forgotten.

Reply:
I agree. I can't remember exactly what I was thinking but it must have been something along the lines of unifying all OQS_KEX_xxx_new functions. It would be much better to update all such non-trivial functions to use an err label.